go/protocol/chat1/extras.go

```diff
@@ -217,6 +217,35 @@ func (m MessageUnboxed) IsValid() bool {
 	return false
 }
 
+// Whether the message is all of:
+// 1. Valid
+// 2. Has a non-deleted body with a type matching the header
+//    (TLFNAME is an exception as it has no body)
+// 3. Supersededby == 0
+func (m MessageUnboxed) IsValidFull() bool {
```

Comment:
I think maybe this should get a better name, its hard to know what is different from the name alone. Also this probably fails `go vet` with the comment.

Reply:
Any ideas for the name? IsValidAndNotDeleted.. but that's kind of not true. IsValidAndWorthyOfDisplay.

Reply:
`IsValidAndDisplayable`?

Reply:
It's true for `DELETE` and `METADATA` so I feel like displayable is a little misleading

Reply:
Also `go vet` doesn't seem to have any problem with this for me

Reply:
We might block it in `protocol`.

Reply:
I'm not sure I understand. Is there a failure somewhere?